src/components/Selector/Selector.tsx

```diff
@@ -1,16 +1,18 @@
 import { Listbox } from "@headlessui/react";
 import { ErrorMessage } from "@hookform/error-message";
-import { FieldValues, Path, get, useController } from "react-hook-form";
+import { FieldValues, get, useController } from "react-hook-form";
 import { OptionType, Props, ValKey } from "./types";
 import { DrawerIcon } from "components/Icon";
 import FocusableInput from "./FocusableInput";
 import { styles, valueKey } from "./constants";
 
-export function Selector<
-  T extends FieldValues,
-  K extends Path<T>,
```

Comment:
Had to remove the 2nd generic parameter to allow for dynamic field names in the bank details form, see how it's [used in RequirementField](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2439/files#diff-c7b73ee138eaa5d5f7f49f7b6fbc0b853f997bfdfdd3718d5209a1fce98666daR50)

Reply:
@misicnenad, since the dynamic use case was only needed in the requirement field, I would have preferrred to just type the path to `any` (as it is not known in advance)
```ts
<Selector<FV, any, string name={dynamicName} />
```
rather than changing all use cases that already benefit from this interface. It's true the `<Field/>` has less strict interface but it doesn't mean it's okay to level into this. 

now mismatches, `FV.usState -> string`, or `<Selector name="city" options={usState}/>` won't be detected anymore which has proved helpful to me when I did some refactoring in registration and widget

I really hope you could bring this strictness level back as the pros of removing it (`declaration a bit shorter to write`) is aesthetic

Reply:
Sure thing, `<Selector<FV, any, string name={dynamicName} />` works fine for bank page needs.

Reverted at https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2439/commits/2ea112e2f9593f208bc1f0e2b432d9335560282b